test/verify/machineslib.py

```diff
@@ -1530,7 +1533,7 @@ class VmDialog:
 
         def __init__(self, test_obj, name=None,
                      sourceType='file', sourceTypeSecondChoice=None, location='',
-                     memory_size=1, memory_size_unit='GiB',
+                     memory_size=256, memory_size_unit='MiB',
```

Comment:
This changes the memory for all tests. While this is a sensible change to reduce the necessary test resources, it should be an independent commit.